app.rb

```diff
@@ -0,0 +1,54 @@
+require "sinatra"
+require "json"
+
+HELP_TEXT = <<~HELP_TEXT.freeze
```

Comment:
What's going on here?

Reply:
The `<<~HELP_TEXT` part is making it a [heredoc](https://infinum.co/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc). The `.freeze` is because it's a constant, and with our Rubocop configs, we want to enforce that it doesn't change.

Reply:
👍